nototools/unittests/font_tests.py

```diff
@@ -330,25 +330,55 @@ def test_format_4_cmap_preset(self):
 
 
 class TestLigatures(FontTest):
-    """Tests formation or lack of formation of ligatures."""
+    """Tests formation or lack of formation of ligatures.
+
+    Must have the following attributes:
+        active: List of (parameters, (sequence, sequence, ...)) tuples, for
+            which each sequence should form a ligature when shaped with the
+            given parameters. The ligature should not be formed by default if
+            parameters are provided.
+        inactive: Similar list of tuples, for sequences which should form a
+            ligature under inverse circumstances.
```

Comment:
Yea, I'm certainly not using "inverse" in a strictly correct sense here. Basically to spell it out I would say:

```
        inactive: List of (parameters, (sequence, sequence, ...)) tuples, for
            which each sequence should not form a ligature when shaped with the
            given parameters. The ligature should be formed by default if
            parameters are provided.
```

but I didn't want to essentially repeat the last attribute description with just the word "not" moved. What if it said "opposite" or "the reverse" circumstances? Or, any other suggestions?